tock/hours/tests/test_forms.py

```diff
@@ -164,16 +172,25 @@ def test_timecard_inline_formset_save_only(self):
         formset.save_only = True
         self.assertTrue(formset.save_only)
 
-    def test_timecard_is_not_100(self):
-        """ Test timecard form data that doesn't total the set working
-        hours """
+    def test_timecard_is_too_big(self):
+        """ Test timecard form data that exceeds the maximum hours allowed """
```

Comment:
Ah okay, I missed the base data in the `setUp` method, thanks for pointing that out!